NodeFirefox/Dockerfile.txt

```diff
@@ -5,17 +5,31 @@ USER root
 #=========
 # Firefox
 #=========
-ENV FIREFOX_VERSION 47.0.1
 RUN apt-get update -qqy \
   && apt-get -qqy --no-install-recommends install firefox \
   && rm -rf /var/lib/apt/lists/* \
-  && wget --no-verbose -O /tmp/firefox.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
-  && apt-get -y purge firefox \
-  && rm -rf /opt/firefox \
-  && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
-  && rm /tmp/firefox.tar.bz2 \
-  && mv /opt/firefox /opt/firefox-$FIREFOX_VERSION \
-  && ln -fs /opt/firefox-$FIREFOX_VERSION/firefox /usr/bin/firefox
+  && apt-get -y purge firefox
+ENV FIREFOX_VERSION 47.0.1
+RUN for VERSION in 48.0.2 47.0.1 46.0.1 45.0.2; do \
```

Comment:
I think it makes sense to keep 47.0.1 for now, as it's the latest version supported without GeckoDriver and is the current version, and I'd like to allow a release of a new docker image without forcing GeckoDriver before Selenium 3 is out of beta.